### ISSUE TITLE
Resolve accessibility compile warning regarding emojis on R&P dashboard

### ISSUE THREAD

**Issue by DanielEFrampton** [PairedStaging/paired-fe#17]
```
 Line 69:15:  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji
```
